content/push/index.textile

```diff
@@ -61,6 +61,26 @@ If you publish via channels, devices or browsers must subscribe to those channel
 
 * The "@clientId@":/push/publish#sub-clientID process subscribes all devices or browsers tied to a particular @clientId@ to a channel in one action, for example the same user's laptop and mobile phone. This approach is useful for subscribing multiple devices or browsers simultaneously.
 
+h2(#lifecycle). Push notification lifecycle
+
+The following diagram shows the push notification lifecycle:
+
+<a href="@content/diagrams/push-lifecycle.png" target="_blank">
+  <img src="@content/diagrams/push-lifecycle.png" style="width: 100%" alt="Push notifications lifecycle">
+</a>
+
+The following explains the push notification lifecycle:
+
+* Publish a push notification:
+** The process begins with your server, which publishes a push notification. This server could be any backend service that needs to send notifications to users' devices.
+** Your server sends the notification data to Ably, a realtime messaging platform.
+* Handle push notification:
+** After receiving the push notification from your server, Ably processes and packages the notification with any additional data payload. This payload may include data such as message content, metadata, or any other relevant information needed for the notification.
+** Ably then forwards the packaged message to the appropriate push notification service _FCM, APNs, or Web Push_.
```

Comment:
web push is not a push notification service, rather a specification which is implemented with different push services depending on the browser.
```suggestion
** Ably then forwards the packaged message to the appropriate push notification service _FCM_, _APNs_, or in the case of Web Push, the browser vendor's notification service.
```